website/templates/emails/pending_retraction_non_admin.txt.mako

```diff
@@ -0,0 +1,7 @@
+Hello ${user.fullname},
+
+I just wanted to let you know, ${initiated_by} has requested a retraction for the following registration: ${registration_link}.
+
+Sincerely yours,
+
+The OSF Robots
```

Comment:
Yes, singular.  We have learned from Terminator that allowing proliferation of robots is not wise.